task_3.py

```diff
@@ -0,0 +1,23 @@
+import math as m
+
+print('Multiparadigm programming languages : Task 3')
+print('Danil Mashkin, FI-9119')
+
+
+def validate_input(var_name):
```

Comment:
I would recommend you to place any functions at the top of the file. (but after imports)